### ISSUE TITLE
typescript: __TypeKind is not imported

### ISSUE THREAD

**Issue by flisky** [dotansimha/graphql-code-generator#1395]
**Describe the bug**
ts compilers complains with `Cannot find name '__TypeKind'.ts(2304)` when there is a introspection field `kind`.

**To Reproduce**
document goes 
```
query userType {
  userType: __type(name: "User") {
    kind
  }
}
```

**Additional context**
I have to add `declare type __TypeKind = typeof import('graphql').__TypeKind;` to fix this problem.

#1380 referenced, and @kamilkisiela, please take a look?

**Comment by kamilkisiela** [comment]
@dotansimha @flisky #1400 should fix it

**Comment by Jonatthu** [comment]
Very nice I will be waiting for this release!

**Comment by dotansimha** [comment]
Fix available in 1.0.0 👍
